draft-amend-tsvwg-multipath-dccp.mkd

```diff
@@ -399,12 +443,12 @@ considerations for:
 
 
 
-* The minimum PMTU of the individual paths must be selected to
-  announce to the application.  Changes of individual path PMTUs
-  must be re-announced to the application if they are lower than the
-  current announced PMTU.
+* The minimum PMTU of the individual paths must be selected to be
+  announced to the application.  Changes of individual path PMTUs
```

Comment:
Would this be better?
"The minimum PMTU of the individual paths must be announced to the application"

Reply:
Incorporated